doc/scripts/videoRecordingExampleJoinUnix.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# A script to generate an ALE video with FFMpeg, *nix systems. 
+
+# -r ## specifies the frame rate
+# -i record/%06d.png indicates we should use sequentially numbered frames in directory 'record'
+# -i sound.wav indicates the location of the sound file
+# -f mov specifies a MOV format
+# -c:a mp3 specifies the sound codec 
+# -c:v libx264 specifies the video codec
+# 
+ffmpeg -r 60 -i record/%06d.png -i record/sound.wav -f mov -c:a mp3 -c:v libx264 agent.mov 
```

Comment:
Ubuntu 14.04 replaced ffmpeg with their own utility 'avconv'. avconv has the exact same syntax as ffmpeg. ffmpeg will be back in upcoming Ubuntu releases, but for now I'd recommend trying both in the bash script. This can be done as follows:

```bash
{
    ffmpeg -r 60 -i record/%06d.png -i record/sound.wav -f mov -c:a mp3 -c:v libx264 agent.mov
} || {
    avconv -r 60 -i record/%06d.png -i record/sound.wav -f mov -c:a mp3 -c:v libx264 agent.mov
}
```